template-concept.md

```diff
@@ -0,0 +1,98 @@
+# Concept Template
+
+Use this template when writing conceptual topics.
+Conceptual topics explain how things work or what things mean.
+They provide helpful context to readers. They do not include procedures.
+
+## Template
+
+The template below includes the standard sections that should appear in conceptual topics, including a topic introduction sentence, an overview, and placeholders for additional sections and subsections.
+Copy and paste the markdown from the template to use it in your topic.
+
+This topic describes...
```

Comment:
Ehhh, maybe? It would look better in preview (not something I can get working for this doc, though) but engineers will just be looking at this in their text editors so I don't know that code tags would be more help than nuisance.

Reply:
I'll add the code tag; it's likely more help than hindrance.

Reply:
I think the code snippets will show where the template begins. I don't think tags will hinder anything. I found it difficult to see where the introduction to the section ended and the template began before the code tags were added.